pages/api/callback.js

```diff
@@ -1,14 +1,23 @@
 import debug from 'debug';
 import * as Sentry from '@sentry/browser';
-
+import base64url from 'base64url';
 import auth0 from '../../lib/auth0';
 
 const dlog = debug('that:api:callback');
 
 export default async function callback(req, res) {
+  const state = base64url.toBuffer(req.query.state || '');
```

Comment:
Could you use the next router here to grab the value you need?

Reply:
It's value which was passed to api/login.js and written to state prior to sending them off to auth0 or authentication. Once you come back to the callback, I am not sure how the router will know where you came from before being sent to auth0

Reply:
I would like to enhance this so that the state value is verified as what was passed in to the callback which isn't happening currently